activiti-spring-conformance/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.activiti</groupId>
+        <artifactId>activiti-core-dependencies</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
+        <relativePath>../activiti-core-dependencies</relativePath>
+    </parent>
+    <artifactId>activiti-spring-conformance</artifactId>
```

Comment:
I'm wondering if it should not have "test" in its name, as it's used for test purpose only.